datarepo-clienttests/build.gradle

```diff
@@ -62,6 +62,13 @@ dependencies {
         implementation "bio.terra:datarepo-client:${datarepoClient}"
     }
 
+    compile group: 'io.kubernetes', name: 'client-java', version: '8.0.2'
+    compile 'com.google.api-client:google-api-client:1.23.0'
+    compile 'com.google.oauth-client:google-oauth-client-jetty:1.23.0'
+    compile 'com.google.apis:google-api-services-people:v1-rev277-1.23.0'
+    compile 'com.google.cloud:google-cloud-bigquery:1.108.1'
+    compile 'com.google.cloud:google-cloud-storage:1.108.0'
+    compile group: 'com.google.auth', name: 'google-auth-library-oauth2-http', version: '0.20.0'
```

Comment:
I think this is a bad merge. Several of these are duplicates of above. Can you just add in the imports you need, and pull out the version number as the ones above do?